webhook/lib/snt/webhook/connection.rb

```diff
@@ -16,6 +16,8 @@ def initialize(options = {})
       end
 
       def request(method, url, params: nil, body: nil, headers: nil)
+        body = body.to_h unless body.nil?
```

Comment:
The call to `body.to_h` will no longer be necessary once you have converted the permitted webhook parameters into a symbolized keys hash in `Admin::WebhooksController`.